### DISCUSSION TITLE
Transformers plugin other models from Hugging face

### DISCUSSION THREAD

**Discussion by vistamou** [explosion/spaCy#9934] [Help: Installation] [answered]
Hello, I want to use another model from hugging face repository, namely this one: https://huggingface.co/DeepPavlov/bert-base-bg-cs-pl-ru-cased, but as I understand (after I tried) it's not possible by just downloading and replacing in the config file: **name =** `path/to/file`--
could you please provide some help ?

**Comment by polm** [comment, marked as answer]
A local file path should work there, though usually we recommend using the name on the Hugging Face Hub - if you use the name the model will be downloaded and set up properly for you. See [here](https://spacy.io/usage/embeddings-transformers#:~:text=You%20can%20also%20use%20a%20local%20file) in the docs. What error are you getting?
